unlock-app/src/utils/durations.js

```diff
@@ -42,6 +42,10 @@ export function durations(seconds, intervals) {
  * @param {number} seconds
  */
 export function durationsAsTextFromSeconds(seconds) {
+  if (seconds < 1) {
+    // 0 or infinite return an empty string
+    return ''
+  }
```

Comment:
This require changing existing test which is failing because we have one for 0.5 seconds.

Reply:
HA! Good point! Let me fix